src/cpu/x64/jit_avx512_core_x8s8s32x_conv_kernel.cpp

```diff
@@ -1301,8 +1300,7 @@ status_t jit_avx512_core_x8s8s32x_fwd_kernel::init_conf(jit_conv_conf_t &jcp,
     if (jcp.is_depthwise) {
         jcp.max_regs_ur = 31 - jcp.is_fast_depthwise - !jcp.is_resrc_depthwise
                 - (jcp.signed_input || jcp.with_input_zp) - (jcp.ver != ver_vnni)
-                - (jcp.signed_input || jcp.with_input_zp || jcp.need_saturation) // both alias
-                - with_quantization;
+                - (jcp.signed_input || jcp.with_input_zp || jcp.need_saturation); // both alias
```

Comment:
max_regs_ur - is calculated correctly. We need to fix (or just comment) the assert itself.

Reply:
Reverted. Assert logic is fixed instead.